internal/testrunner/runners/system/servicedeployer/custom-agent-base-config.yml

```diff
@@ -0,0 +1,15 @@
+version: "2.3"
+services:
+  custom-agent:
+    image: "${ELASTIC_AGENT_IMAGE_REF}"
+    healthcheck:
+      test: "elastic-agent status"
+      retries: 180
+      interval: 1s
+    hostname: custom-agent
+    environment:
+      - FLEET_ENROLL=1
+      - FLEET_INSECURE=1
+      - FLEET_URL=http://fleet-server:8220
```

Comment:
Based on this [file](https://github.com/elastic/elastic-package/blob/main/internal/profile/_static/elastic-agent_8x.env), isn't `FLEET_TOKEN_POLICY_NAME` not required?

Reply:
Would only be needed if we want it to be enrolled to the current stack policy, but we are just interested in it during the test, so the policy will change right after for the test one, running the config useless.

Reply:
Ah, you're right! Could you please drop a comment there in case somebody else will come here.